lib/cartodb/api/configuration.rb

```diff
@@ -0,0 +1,69 @@
+module CartoDB
+  module Api
+    class Configuration
+
+      PROTOCOL = 'https'
+      DOMAIN = 'cartodb.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.